source/MaterialXTest/ShaderValid.cpp

```diff
@@ -77,7 +77,7 @@ TEST_CASE("GLSL Source", "[shadervalid]")
         validator->initialize();
         validator->setImageHandler(handler);
         // Set geometry to draw with
-        const std::string geometryFile(mx::FilePath::getCurrentPath().asString() + "/documents/TestSuite/Geometry/sphere.obj");
+        const std::string geometryFile(mx::FilePath::getCurrentPath().asString() + "documents/TestSuite/Geometry/sphere.obj");
```

Comment:
Why not use mx::FilePath::getCurrentPath() / mx::FilePath("documents/TestSuite/Geometry/sphere.obj"); ?

Reply:
Old code :) I'll clean this up.